website/content/docs/secrets/databases/custom.mdx

```diff
@@ -207,8 +207,7 @@ func Run() error {
 ## Running your plugin
 
 The above main package, once built, will supply you with a binary of your
-plugin. We also recommend if you are planning on distributing your plugin to
-build with [gox](https://github.com/mitchellh/gox) for cross platform builds.
+plugin.
```

Comment:
Might not include this bit, our removal of gox from vault doesn't necessarily have implications for plugins.

Reply:
:+1: